File: BatteryLife.java
import java.util.Scanner;

public class BatteryLife {
    private double capacity;

    public BatteryLife(double initialCapacity) {
        capacity = initialCapacity;
    }

    public void discharge(double usage) {
        capacity -= usage;
    }

    public void charge(double amount) {
        capacity += amount;
    }

    public double getPercentage(double maxCapacity) {
        return (capacity / maxCapacity) * 100;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter initial capacity of the battery: ");
        double initialCapacity = scanner.nextDouble();
        
        System.out.print("Enter maximum capacity of the battery: ");
        double maxCapacity = scanner.nextDouble();

        BatteryLife battery = new BatteryLife(initialCapacity);

        System.out.println("Battery Percentage: " + battery.getPercentage(maxCapacity));

        scanner.close();
    }
}

File: battery_life.py
class Battery:
    def __init__(self, initial_capacity):
        self.capacity = initial_capacity

    def discharge(self, usage):
        self.capacity -= usage

    def charge(self, amount):
        self.capacity += amount

    def get_percentage(self, max_capacity):
        return (self.capacity / max_capacity) * 100

# Example usage:
if __name__ == "__main__":
    initial_capacity = float(input("Enter initial capacity of the battery: "))
    max_capacity = float(input("Enter maximum capacity of the battery: "))

    battery = Battery(initial_capacity)

    print("Battery Percentage:", battery.get_percentage(max_capacity))
